client/src/components/share/share-template.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { faTwitter } from '@fortawesome/free-brands-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ShareRedirectProps } from './types';
+
+export const ShareTemplate: React.ComponentType<ShareRedirectProps> = ({
+  redirectURL
+}) => {
+  const { t } = useTranslation();
+  return (
+    <a
+      title='Share on Twitter'
+      data-testid='ShareTemplateWrapperTestID'
+      className='btn fade-in'
+      href={redirectURL}
+      target='_blank'
+      rel='noreferrer'
+    >
+      <span className='sr-only'>opens in new window</span>
+      <span className='sr-only'>{t('share-on-twitter')}</span>
```

Comment:
```suggestion
      <span className='sr-only'>{t('buttons.share-on-twitter')}, {t('aria.opens-new-window')}</span>
```

"Opens in new window" should come after the link text and it already has a translation we should use.

Also, this suggestion is based on how the link currently looks. If we decide to display "Twitter" next to the link then this will change a little.

Reply:
Actually this text is surrounded by link

Reply:
@BalanaguYashwanth I'm not sure what your last comment is trying to say. When a screen reader hears this link it should say the name of the button first and then "opens in new window". That's the change I suggested.

Reply:
ok got it now